scss/_navbar.scss

```diff
@@ -115,6 +115,12 @@
   @include hover-focus {
     text-decoration: none;
   }
+
+  &::before {
+    font-weight: 500;
+    content: "\2261"; // see http://unicode.johnholtripley.co.uk/2261/
```

Comment:
Maybe comment it as `IDENTICAL TO (U+2261)` similar to https://github.com/twbs/bootstrap/blob/1347cb0152c992f8b71e185fa7ef1794fc94d931/scss/_carousel.scss#L145?